lib/util.js

```diff
@@ -151,7 +152,24 @@ module.exports.readFilesToArray = function(fileList) {
   var contentArray = [];
   fileList.forEach(function(filePath) {
     try {
-      contentArray.push(fs.readFileSync(filePath));
+      let extension = filePath.split('.').pop();
+      let content = fs.readFileSync(filePath);
+      if(extension == 'p12' || extension == 'pfx'){
+        let p12Der = forge.util.decode64(content.toString('base64'));
+        let p12Asn1 = forge.asn1.fromDer(p12Der);
+        let p12;
+        try{
+          p12 = forge.pkcs12.pkcs12FromAsn1(p12Asn1, "");
+        } catch (e) {
+          loggers.bootstrapLogger.warn("ZWED0173W", e.message);
+          p12 = forge.pkcs12.pkcs12FromAsn1(p12Asn1, process.env.KEYSTORE_PASSWORD || "");
+        }
+        let certData = p12.getBags({bagType: forge.pki.oids.certBag});
+        let certBag = certData[forge.pki.oids.certBag][0];
```

Comment:
Why should we read only the first certificate in the chain and ignore the remaining?